src/modules/crewMethods.ts

```diff
@@ -67,9 +78,52 @@ export const crewMethods = {
             method: 'POST',
             headers: {
                 authorization: apiToken,
+                'Content-Type': 'application/json',
             },
             body: JSON.stringify({ search, take: take || defaultListLimit, filter }),
         });
         return getDataFromResponse<Group[]>(response);
     },
+
+    getUserInfo: async (email: string) => {
+        const { url, apiToken } = checkConfig();
+
+        const params = new URLSearchParams({ field: 'email', value: email });
+        const response = await fetch(`${url}/api/rest/users/get-by-field?${params}`, {
+            method: 'GET',
+            headers: {
+                authorization: apiToken,
+            },
+        });
+        return getDataFromResponse<CrewUser>(response);
+    },
+
+    getAchievements: async (data: GetAchievements) => {
+        const { search, take } = data;
+        const { url, apiToken } = checkConfig();
+        const params = new URLSearchParams();
+        if (search) params.append('search', search);
+        if (take) params.append('take', String(take));
+
+        const response = await fetch(`${url}/api/rest/achievements/list?${params}`, {
+            method: 'GET',
+            headers: {
+                authorization: apiToken,
+            },
+        });
+        return getDataFromResponse<Achievement[]>(response);
+    },
+
+    giveAchievement: async (data: GiveAchievement) => {
+        const { url, apiToken } = checkConfig();
+        const response = await fetch(`${url}/api/rest/achievements/give`, {
+            method: 'POST',
+            headers: {
+                authorization: apiToken,
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(data),
```

Comment:
don't you want to create a wrapper for these fetches? it's a copypaste

Reply:
not really: all of these fetches have `authorization: apiToken,` but almost half of them are GET, so they don't have content type and body

Reply:
or maybe wrappers for get and wrapper for post apart?